test/remote/gateways/remote_ogone_test.rb

```diff
@@ -74,6 +92,55 @@ def test_successful_purchase_with_3d_secure
     assert_equal '46', response.params['STATUS']
     assert_equal OgoneGateway::SUCCESS_MESSAGE, response.message
     assert response.params['HTML_ANSWER']
+    assert_match 'mpi-v1', Base64.decode64(response.params['HTML_ANSWER'])
+  end
+
+  def test_successful_purchase_with_3d_secure_v2
+    assert response = @gateway.purchase(@amount, @credit_card_d3d_2_challenge, @options_browser_info.merge(d3d: true))
+    assert_success response
+    assert_equal '46', response.params['STATUS']
+    assert_equal OgoneGateway::SUCCESS_MESSAGE, response.message
+    assert response.params['HTML_ANSWER']
+    assert_match 'mpi-v2', Base64.decode64(response.params['HTML_ANSWER'])
+  end
+
+  def test_successful_purchase_with_3d_secure_v2_friction_less
```

Comment:
💬  Comment:

 "friction less" is a single word

Reply:
ups, thanks, I'll fix that